src/hooks/dialogmote/document/useDialogmoteDocumentComponents.ts

```diff
@@ -9,46 +9,59 @@ import { tilDatoMedUkedagOgManedNavnOgKlokkeslett } from "@/utils/datoUtils";
 import { genererDato } from "@/sider/mote/utils";
 import { DocumentComponentDto } from "@/data/documentcomponent/documentComponentTypes";
 import { useDocumentComponents } from "@/hooks/useDocumentComponents";
+import { Malform } from "@/context/malform/MalformContext";
 
 export const useDialogmoteDocumentComponents = () => {
   const { getHilsen, getIntroGjelder, getIntroHei } = useDocumentComponents();
   const { getCurrentNarmesteLeder } = useLedereQuery();
 
   const getVirksomhetsnavn = (
-    virksomhetsnummer: string | undefined
+    virksomhetsnummer: string | undefined,
+    malform?: Malform
```

Comment:
Her hadde det feks vært bedre å kalle hooken og bruke den i stedet for å ta inn `malform` som parameter, men ettersom det foreløpig ikke er målform på alle dialogmøte-tekstene så må vi ta det litt steg for steg på denne måten.